### PR TITLE
[MM-58518] Migrate tooltips of "components/copy_text.tsx" to WithTooltip

### PR DESCRIPTION
**Pull request by Gesare5**
#### Summary

Replaces Tooltip and Overlay Trigger components with the WithTooltip component in copy_text.tsx

#### Ticket Link

Fixes [#27228](https://github.com/mattermost/mattermost/issues/27228)
Jira [https://mattermost.atlassian.net/browse/MM-58518]

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/components/copy_text.tsx [hunk 1]
```diff
@@ -2,14 +2,12 @@
...
-import {FormattedMessage} from 'react-intl';
+import {useIntl} from 'react-intl';
 
-import OverlayTrigger from 'components/overlay_trigger';
-import Tooltip from 'components/tooltip';
-
-import Constants from 'utils/constants';
 import {copyToClipboard} from 'utils/utils';
 
+import WithTooltip from './with_tooltip';
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/copy_text.tsx:9]: instead of relative importing can you use absolute like `import X from 'components/x'

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

**Comment by M-ZubairAhmed** [comment]
@Gesare5 apologies would you be able to work on https://github.com/mattermost/mattermost/issues/27156 i noticed there are some changes which might be needed in this component.

**Comment by Gesare5** [comment]
@M-ZubairAhmed No problem

**Comment by M-ZubairAhmed** [comment]
Closing in favour of https://github.com/mattermost/mattermost/pull/27413

**Pull request closed without merging**